### ISSUE TITLE
standard version scheme

### ISSUE THREAD

**Issue by mr-c** [amplab/snap#57]
Hello,

I'm packaging SNAP for Debian. I was wondering if you could number your releases using a standard scheme like http://semver.org/

"1.0beta.18" doesn't fit any scheme I know of and is hard to parse by the upstream watcher that alerts when there is a new release available.

Thanks!

**Comment by mr-c** [comment]
FYI, for Debian I re-versioned snap-aligner as "0.18~1.0~beta.18" to indicate that it was a successor to the 0.14, 0.15 series, and was a beta for the 1.0 release. Otherwise a future "1.1" release would still be seen as inferior to "1.0beta.18".

**Comment by bolosky** [comment]
We should continue using a standard scheme going forward from v1.0.0.

**Comment by mr-c** [comment]
@bolosky Congratulations on the v1.0 release!

**Comment by bolosky** [comment]
Thanks!  It took a lot of work to get here.
